apps/jira/functions/package.json

```diff
@@ -3,16 +3,6 @@
   "version": "1.4.62",
   "main": "index.js",
   "private": true,
-  "scripts": {
```

Comment:
Why is this deleted?

Reply:
I suspect a rebase to master, I've reinstated the script-section.